models/app.models.js

```diff
@@ -0,0 +1,10 @@
+const db = require("../db/connection");
+
+exports.selectCategories = () => {
+  let categorySQLStr = `
+    SELECT * FROM categories
+    ;`;
+  return db.query(categorySQLStr).then((categories) => {
+    return categories.rows;
```

Comment:
Nice that you send back categories.rows here so you can rename in the controller, good stuff